tests/rust/test_sandbox.py

```diff
@@ -2,7 +2,7 @@
 import errno
 import os
 import platform
-from ctypes import c_uint32
+import threading
```

Comment:
Can you explain a bit more in the commit about what's happening here ? Are landlock limits applied to the thread only and not the main thread ?

Reply:
added comment to the test